.github/workflows/main.yml

```diff
@@ -17,7 +17,6 @@ jobs:
         os:
           - ubuntu-20.04
           - ubuntu-22.04
-          - windows-2019
```

Comment:
The Python version in this runner - https://github.com/actions/runner-images/blob/main/images/windows/Windows2019-Readme.md#language-and-runtime - does support support recent versions of cfn-lint https://github.com/ScottBrenner/cfn-lint-action/actions/runs/10277441399/job/28439493287?pr=420#step:3:33